web/api/legacy/query.go

```diff
@@ -61,6 +61,12 @@ func parseDuration(d string) (time.Duration, error) {
 	return time.Duration(dFloat * float64(time.Second/time.Nanosecond)), nil
 }
 
+// Options handles OPTIONS requests to /api/... endpoints.
```

Comment:
All of them?
On Jan 26, 2016 01:52, "Tobias Schmidt" <notifications@github.com> wrote:

> In web/api/legacy/query.go
> <https://github.com/prometheus/prometheus/pull/1347#discussion_r50782092>:
>
> > @@ -61,6 +61,12 @@ func parseDuration(d string) (time.Duration, error) {
> >  	return time.Duration(dFloat * float64(time.Second/time.Nanosecond)), nil
> >  }
> >
> > +// Options handles OPTIONS requests to /api/... endpoints.
>
> Which /api/... endpoints?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/1347/files#r50782092>.
>